Documentation/project-docs/developer-guide.md

```diff
@@ -23,12 +23,11 @@ The tasks are represented in scripts (cmd/sh) in the root of the repo:
 * Clean
 * Sync
 * Build
-* Run Tests
-* Publish packages
+* Build-Tests
```

Comment:
If they are logical steps, then we need to express that Build-Tests covers also Run-Tests.